ui/StatusQ/src/StatusQ/Controls/StatusInput.qml

```diff
@@ -435,7 +417,6 @@ Item {
     }
 
     implicitWidth: 448
```

Comment:
What about removing that as well? It's not real implicit width, if given width is desired, it should be set externally.